src/material/sort/sort-header.scss

```diff
@@ -31,6 +31,7 @@ $mat-sort-header-arrow-hint-opacity: 0.38;
   outline: 0;
   font: inherit;
   color: currentColor;
+  position: relative;
```

Comment:
Maybe add a comment here and the other places that got `position: relative` that mentions why we need it.